maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,21 @@
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0px 10px 13px -7px #000000, 0px 7px 5px 0px rgba(125,125,125,0.43);
```

Comment:
Use color constants for `#0000` and `rgba(....)`

I also dont think you need so many parameters here, the 3 first and a color should be enought